codegen/src/main/twirl/templates/JavaCommon/ApiInterface.scala.txt

```diff
@@ -20,6 +20,7 @@ public interface @{service.name} {
   }
 
   static String name = "@service.grpcName";
+  static akka.grpc.ServiceObject service = new akka.grpc.ServiceObjectImpl(name, @service.descriptor);
```

Comment:
Are these final automatically because they are in an interface? (if not, they probably should be)

Reply:
checked with javap, they're indeed final